### PR TITLE
docs: blog about labs initiative

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <rawlingsj80@gmail.com>

### CODE REVIEW

File: content/en/blog/news/labs-202002.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- second year was about improving that developer experience and making the architectural approaches we settled on a reliable experience.  This has been a lot of work and there's still a lot more to do, infact we know this will always be an ongoing effort.  Some of the recent highlights of this effort can start to be seen with more details on this [blog](/blog/2020/02/26/cjxd-feb-20/)
+
+- now that we have started our third year we look at how we can effectively innovate in Jenkins X, grow collaborations with other open source projects and communities and of course build up the Jenkins X community itself.  All the time we need to do this with the second years efforts in mind and not loose sight that we need to evolve as a mature project that people and teams can rely on.
```
Comment (deanesmith) [inline content/en/blog/news/labs-202002.md]: Should be "lose"?

Comment (rawlingsj) [reply content/en/blog/news/labs-202002.md]: good catch! thanks :)

**Pull request merged**